deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/EnvCapacities.java

```diff
@@ -157,9 +158,12 @@ public void add(
             @NotEmpty String name,
             @Context SecurityContext sc)
             throws Exception {
+        name = name.replace("\"", "");
```

Comment:
Should we do this to all endpoints? Or we might have inconsistent behaviors.

Reply:
NVM

Reply:
This line was just moved from below, so kept for backward compatibility